### PR TITLE
radare2 0.9.9

### PR DESCRIPTION
**Pull request by cbarcenas**
Bump radare2 to version 0.9.9

### CODE REVIEW

**Comment by bfontaine** [comment]
The build is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/29443/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_radare2/

File: Library/Formula/radare2.rb [hunk 1]
```diff
@@ -39,14 +39,23 @@ class Radare2 < Formula
...
+    rm_rf "shlr/capstone"
+
     # Build Radare2 before bindings, otherwise compile = nope.
-    system "./configure", "--prefix=#{prefix}", "--with-openssl"
-    system "make"
-    system "make", "install"
+    system "./sys/install.sh"
 
     resource("bindings").stage do
-      ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
```
Comment (otaku) [inline Library/Formula/radare2.rb]: I needed to add this back, otherwise it could not find r_core

**Comment by DomT4** [comment]
Ping.

**Comment by DomT4** [comment]
Version bump was merged in a more recent PR.

**Pull request closed without merging**